.gitignore

```diff
@@ -59,4 +59,7 @@ buck-out/
 /ios/Pods/
 
 # eslintignore
-.eslintignore
\ No newline at end of file
+.eslintignore
+
+# Google API Token
+/app/components/token.js
```

Comment:
I guess when this is pushed you'll pass this token around and it will live in our local.